.rubocop.yml

```diff
@@ -20,9 +24,27 @@ Metrics/BlockLength:
     - 'spec/**/*.rb'
     - 'openfeature-sdk.gemspec'
 
+Gemspec/DevelopmentDependencies:
+  EnforcedStyle: gemspec
```

Comment:
I typically don't use the `development_dependency` style and favor using `Gemfile`, but I opted to use what the project already uses.